_posts/en/newsletters/2021-12-22-newsletter.md

```diff
@@ -0,0 +1,665 @@
+[implemented][bitcoin core #7600] in Bitcoin Core since 2016, there has
+so far been no way for nodes to relay transactions as packages, meaning
+low-feerate parent transactions might not reach miners during
+high-feerate periods even if they have high-feerate children.  This
+makes [CPFP fee bumping][topic cpfp] unreliable for contract protocols
+using presigned transactions, such as LN.  Package relay hopes to solve
+this key safety issue.
+
+An idea original proposed in 2019 for LN saw renewed life in June.  The
+original *fast forwards* idea described how an LN wallet could receive
+or relay a payment with fewer network round-trips, reducing network
+bandwidth and payment latency.  The idea was [expanded][ff expanded]
+this year to describe how an LN wallet could receive multiple payments
+without bringing its signing key online for every payment, making it
+easier to keep that signing key secured.
+
+## July
+
+After years of discussion and development, the first implementation of a
+decentralized liquidity advertisements system was [merged][cl#4639] into
+an LN implementation.  The still-draft [liquidity ads][bolts #878]
+proposal allows a node to use the LN gossip protocol to advertise its
+willingness to lease out its funds for a period of time, giving other
+nodes the ability to buy incoming capacity that allows them to receive
+instant payments. A node that sees the advertisement can simultaneously
+pay for and receive the incoming capacity using a [dual funded][topic
+dual funding] channel open. Although there’s no way to enforce that the
+advertising node actually routes payments, the proposal does incorporate
+an earlier proposal also [later used][lnd#5709] in Lightning Pool that
+prevents the advertiser from using their money for other purposes until
+the agreed upon lease period has concluded.  That means refusing to
+route payments would provide no advantages but would deny the
+advertising node the opportunity to earn routing fees.
+
+<!-- TODO:EXTRA:table comparing liquidity ads and Pool -->
+
+Three years after [first being proposed][descriptor gist] for Bitcoin
+Core, [draft BIPs][descriptor bips1] were [created][descriptor bips2] for
+[output script descriptors][topic descriptors].  Descriptors are strings
+that contain all the information necessary to allow a wallet or other
+program to track payments made to or spent from a particular script or
+set of related scripts (i.e. an address or a set of related addresses
+such as in an HD wallet).  Descriptors combine well with [miniscript][topic miniscript] in
+allowing a wallet to handle tracking and signing for a larger variety of
+scripts.  They also combine well with [PSBTs][topic psbt] in allowing
+the wallet to determine which keys it controls in a multisig script.  By
+the end of the year, Bitcoin Core had made descriptor-based wallets its
+[default][descriptor default] for newly-created wallets.
+
+A common way of opening LN channels that had never before been official
```

Comment:
s/official/officially

Reply:
Wasn't it "officially supported" by one LN implementation? Perhaps instead of "officially supported" it could be described as previously "not part of the LN specification".